lib/cn/csched_sp3.h

```diff
@@ -35,7 +35,7 @@ struct sp3_node {
     struct sp3_rbe   spn_rbe[wtype_MAX];
     struct list_head spn_rlink;
     struct list_head spn_alink;
-    bool             spn_initialized;
+    bool             spn_managed;
```

Comment:
Any comment need updating?

Reply:
I could write a treatise, but the next guy should have to suffer like I did ;-)